examples/api/lib/services/system_chrome/system_chrome.set_system_u_i_overlay_style.1.dart

```diff
@@ -9,30 +9,31 @@ import 'dart:math' as math;
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
-void main() => runApp(const MyApp());
+void main() => runApp(const SystemOverlayStyleApp());
```

Comment:
```suggestion
// Flutter code sample for setting the [SystemUiOverlayStyle] with an [AnnotatedRegion].

void main() => runApp(const SystemOverlayStyleApp());
```

Reply:
Did you mean to replace the comment on line 5 with this new one?

Reply:
For the moment, I updated the PR and put this comment on line 5.

Reply:
Oh yes that is what I meant, sorry. Thanks!